apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -32,10 +81,123 @@ for (const [i, transaction] of transactions.entries()) {
       await app.account.clickSend();
       await app.send.fillTxInfo(transaction);
       await app.send.expectTxInfoValidity(transaction);
-      await app.send.clickContinue();
+      await app.send.clickContinueToDevice();
 
       await app.speculos.expectValidTxInfo(transaction);
       await app.send.expectTxSent();
+      await app.account.expectTxInfos(transaction);
+
+      await app.layout.goToAccounts();
+      await app.accounts.navigateToAccountByName(transaction.accountToCredit.accountName);
+      await app.account.expectReceiver(transaction);
+    });
+  });
+}
+
+for (const [i, transaction] of transactionsToken.entries()) {
+  test.describe.parallel("Send token", () => {
```

Comment:
can you please make test names more clear? because currently, in this precise test you check receive address filed validation, but from the current name one can get impression that it's full e2e flow